.ci/RELEASE_NOTES_GUIDE.md

```diff
@@ -117,12 +117,33 @@ Starting on Nov 1, 2019, Cloud Functions API will be private by default. Add app
 
 Don’t write notes like:
```

Comment:
Can you also add an example of something that tells what was done instead of what the user sees from it? Some  examples that come to mind are things like PATCH vs UPDATE (likely actually a bugfix) or "resource x is now autogenerated" (probably no impact, but maybe fields have changed or there are id/import format changes)